bin/packages/get-babel-config.js

```diff
@@ -23,7 +23,6 @@ const babelConfigs = {
 	main: Object.assign( {}, babelDefaultConfig, {
 		plugins,
 		presets: [
-			[ '@babel/preset-typescript' ],
```

Comment:
This preset is shipped with the wordpress preset so duplicating it here causes warnings